RocketChat/Rocket.Chat
# [FIX] CSV Importer failing to import users

## Proposed changes (including videos or screenshots)


- Update use of `setRealName` function to `_setRealName`.



## Issue(s)

Closes #24073
Fixes #23997

## Steps to test or reproduce

1.  Create a CSV file (e.g. `users.csv`) with the following content:

```plain
bradley.hilton,bradley.hilton@example.com,Bradley Hilton
billy.bob, billy.bob@example.com, Billy Bob Jr.
graywolf336,graywolf336@example.com,GrayWolf336
```

2. Create a ZIP file containing only the CSV file you created in the previous step;

3. Go to **Administration > Import > Import New File**, then choose **CSV as the Import Type** and upload the **ZIP** file from your computer;

4. Click on Import. **_Expected behavior:_** 3 users should be displayed with all the info given in the CSV file.

5. Click on Start Importing. **_Expected behavior:_** all 3 users should be succesfully imported **_with their corresponding real names_** (what can be checked in **Administration > Users**);

6. Delete the 3 users and repeat this process using a CSV with a single user. No error should be displayed in the whole process.

## Further comments
